onnxscript/rewriter/generic_pattern.py

```diff
@@ -294,6 +294,78 @@ def make_node(
         return proto
 
 
+class PatternMatchResult:
+    """Stores information about a match if a match was successful."""
+
+    def __init__(
+        self,
+        pattern: "GenericPattern",
```

Comment:
```suggestion
        pattern: GenericPattern,
```

Reply:
Class GenericPattern is defined after this line.

Reply:
That's ok. `from __future__ import annotations` will quote this for us so we don't have to.

Reply:
done